### PR TITLE
moved wrapper to sit inside _app

### PR DESCRIPTION
**Pull request by ARADDCC002**

### CODE REVIEW

File: frontend/pages/_app.tsx [hunk 1]
```diff
@@ -41,7 +42,9 @@ export default function MyApp(props: AppProps) {
             <SnackbarProvider>
               <LocalizationProvider dateAdapter={AdapterDayjs} adapterLocale='en-gb'>
                 <CssBaseline />
-                <Component {...pageProps} />
+                <Wrapper {...pageProps}>
```
Comment (ARADDCC012) [inline frontend/pages/_app.tsx:45]: Don't think we need to pass `pageProps` to `Wrapper`

File: frontend/pages/help.tsx [hunk 2]
```diff
@@ -24,7 +24,8 @@ export default function Help() {
   if (error) return error
 
   return (
-    <Wrapper title='Help' page='help'>
+    <>
+      <Title title='Help' />
       {isUiConfigLoading && <Loading />}
       {uiConfig && (
         <Paper sx={{ py: 5, px: 5 }}>
```
Comment (ARADDCC012) [inline frontend/pages/help.tsx:31]: I think we need a `<Container maxWidth='xl'>` wrapping this

File: frontend/pages/docs/api.tsx [hunk 3]
```diff
@@ -3,17 +3,17 @@ import 'swagger-ui-react/swagger-ui.css'
...
-import Wrapper from '../../src/Wrapper'
+import Title from 'src/common/Title'
 
 const SwaggerUI = dynamic(import('swagger-ui-react'), { ssr: false })
 
 export default function API() {
   return (
-    <Wrapper title='API Documentation' page='api'>
+    <>
+      <Title title='API' />
```
Comment (ARADDCC012) [inline frontend/pages/docs/api.tsx:13]: I think we need a `<Container maxWidth='xl'>` under here

File: frontend/pages/model/[modelId].tsx [hunk 4]
```diff
@@ -67,7 +67,8 @@ export default function Model() {
   if (error) return error
 
   return (
-    <Wrapper title={model ? model.name : 'Loading...'} page='marketplace' fullWidth>
+    <>
+      <Title title={model ? model.name : 'Loading...'} />
       {(isModelLoading || isUiConfigLoading) && <Loading />}
       {model && (
         <PageWithTabs
```
Comment (ARADDCC012) [inline frontend/pages/model/[modelId].tsx:74]: I think we need a `<Container maxWidth='xl'>` wrapping this

File: frontend/pages/review.tsx [hunk 5]
```diff
@@ -14,8 +14,9 @@ export default function Review() {
   )
 
   return (
-    <Wrapper title='Review' page='review' fullWidth>
+    <>
+      <Title title='Review' />
       <PageWithTabs title='Your Reviews' tabs={tabs} />
```
Comment (ARADDCC012) [inline frontend/pages/review.tsx:19]: I think we need a `<Container maxWidth='xl'>` wrapping this

File: frontend/pages/schemas/list.tsx [hunk 6]
```diff
@@ -1,14 +1,15 @@
...
+import Title from 'src/common/Title'
 import SchemaTab from 'src/schemas/SchemaTab'
-import Wrapper from 'src/Wrapper'
 
 export default function SchemasPage() {
   return (
-    <Wrapper title='Schemas' page='beta/schemas' fullWidth>
+    <>
+      <Title title='Schemas' />
       <Schemas />
```
Comment (ARADDCC012) [inline frontend/pages/schemas/list.tsx:11]: I think we need a `<Container maxWidth='xl'>` wrapping this

File: frontend/pages/settings/index.tsx [hunk 7]
```diff
@@ -27,9 +27,10 @@ export default function Settings() {
   if (error) return error
 
   return (
-    <Wrapper fullWidth title='Settings' page='settings'>
+    <>
+      <Title title='Settings' />
       {isCurrentUserLoading && <Loading />}
       <PageWithTabs title='Settings' tabs={tabs} />
```
Comment (ARADDCC012) [inline frontend/pages/settings/index.tsx:33]: I think we need a `<Container maxWidth='xl'>` wrapping this

File: frontend/src/Wrapper.tsx [hunk 8]
```diff
@@ -69,9 +66,6 @@ export default function Wrapper({ title, page, children, fullWidth = false }: Wr
 
   return (
     <ThemeProvider theme={theme}>
```
Comment (ARADDCC012) [inline frontend/src/Wrapper.tsx:68]: `ThemeProvider` can be removed as it's already in `_app.tsx`

File: frontend/src/Wrapper.tsx [hunk 9]
```diff
@@ -69,9 +66,6 @@ export default function Wrapper({ title, page, children, fullWidth = false }: Wr
...
-      <Head>
-        <title>{`${title} :: Bailo`}</title>
-      </Head>
       <Banner />
       <Box sx={{ display: 'flex' }}>
         <CssBaseline />
```
Comment (ARADDCC012) [inline frontend/src/Wrapper.tsx:71]: `CssBaseline` can be removed as it's already in `_app.tsx`

File: frontend/src/Wrapper.tsx [hunk 10]
```diff
@@ -80,7 +74,7 @@ export default function Wrapper({ title, page, children, fullWidth = false }: Wr
           <>
             <TopNavigation drawerOpen={open} pageTopStyling={pageTopStyling} currentUser={currentUser} />
             <SideNavigation
-              page={page}
+              page={router.route.split('/')[1].replace('/', '')}
```
Comment (ARADDCC012) [inline frontend/src/Wrapper.tsx:77]: Consider memoising this
```
const page = useMemo(() => router.route.split('/')[1].replace('/', ''), [router])
```

File: frontend/src/common/Title.tsx [hunk 11]
```diff
@@ -0,0 +1,13 @@
+import Head from 'next/head'
+
+export interface TitleProps {
+  title: string
```
Comment (ARADDCC012) [inline frontend/src/common/Title.tsx:4]: Consider renaming to `text` so we don't have a prop with the same name as the component

File: frontend/src/wrapper/SideNavigation.tsx [hunk 12]
```diff
@@ -117,7 +117,7 @@ export default function SideNavigation({
               selectedPage={page}
               primaryText='Marketplace'
               drawerOpen={drawerOpen}
-              menuPage='marketplace'
+              menuPage='model'
```
Comment (ARADDCC012) [inline frontend/src/wrapper/SideNavigation.tsx:120]: This should match when we're at `<baseUrl>/`, not when we're at `<baseUrl>/model`. Might be as simple as making this an empty string.

**Review by ARADDCC012 — CHANGES_REQUESTED** [review]

File: frontend/pages/settings/index.tsx (hunk 7 above)
Comment (ARADDCC002) [reply frontend/pages/settings/index.tsx:33]: Container's give a padding on both height and width, whereas we want any page with the tabbed view _not_ to have padding on the width.

Comment (ARADDCC012) [reply frontend/pages/settings/index.tsx:33]: You're entirely right, I thought I'd removed all the comments on tab views. My mistake!

File: frontend/src/docs/DocsWrapper.tsx [hunk 13]
```diff
@@ -110,6 +111,7 @@ export default function DocsWrapper({ children }: DocsWrapperProps): ReactElemen
   return (
     <>
       {/* Banner height + Toolbar height = 96px */}
```
Comment (ARADDCC012) [inline frontend/src/docs/DocsWrapper.tsx:113]: This comment should be above the `Box` below as it explains the height calc

**Review by ARADDCC012 — APPROVED** [review]

**Pull request merged**